test/runtests.jl

```diff
@@ -112,18 +118,24 @@ else
     @test start(t) == nothing
     @test write(t, round.(UInt8, [1 0; 0 0; 1 0; 0 1; 1 1; 0 1])) == 6
     @test stop(t) == nothing
-    @test NIDAQ.CfgSampClkTiming(t.th, convert(Ref{UInt8},b""), 100.0, NIDAQ.Val_Rising,
+    rslt = Ref{UInt32}(0)
+    NIDAQ.DAQmxGetBufInputOnbrdBufSize(t.th, rslt)
```

Comment:
should be `DAQmxGetBufOutputOnbrdBufSize`, not `...Input...`.  otherwise, this works on my box.  did you get a chance to test on yours?